docs/howto/create-a-release.rst

```diff
@@ -0,0 +1,105 @@
+###########################
+Creating a release for Cylc
+###########################
+
+
+Before you start (First time)
+=============================
+* Create a GPG key for verification on Github: `Instructions <https://help.github.com/en/articles/generating-a-new-gpg-key>`_
+  At step 3 I had to use ``gpg --default-key rsa4096 --gen-key``.
+
+* `Share this key locally with git, as well as Github: <https://help.github.com/en/articles/telling-git-about-your-signing-key>`_
+
+* Create a PyPI account for yourself if you don't already have one.
+  `Sign up page <https://www.google.com/url?sa=t&rct=j&q=&esrc=s&source=web&cd=1&cad=rja&uact=8&ved=2ahUKEwjouIq44MrjAhUKTsAKHeeoCyQQFjAAegQIAxAB&url=https%3A%2F%2Fpypi.org%2Faccount%2Fregister%2F&usg=AOvVaw2ZnHHcXEbkC1c5UJPPMg5a>`_
```

Comment:
Is it the right URL?

Reply:
Main use of private browsing: Ensure that I see what a new user would see when writing documentation...